src/http/Request.php

```diff
@@ -127,10 +127,17 @@ class Request implements RequestInterface {
   /**
    * Holds the parsed body.
    *
+   * @var array
```

Comment:
this file seems to be duplicated

``src/http/Request.php`` vs ``src/Http/Request.php``